### DISCUSSION TITLE
Textbox on Grid toolbar.

### DISCUSSION THREAD

**Discussion by juliermefelix** [serenity-is/Serenity#6153] [Help] [answered]
Esteemed,
how to insert a textbox in the grid toolbar. i can insert new buttons but textbox not.
similar to the search textbox

![image](https://user-images.githubusercontent.com/7516443/146829108-e144d043-33cf-4095-82d1-d743041a7b58.png)

**Comment by wezmag** [comment, marked as answer]
You can add textbox using jQuery in the constructor of the grid.ts file.
However, you will need to add some css to make it looks better.
```
this.toolbar.element.append('<input type="text" />');
```

**Reply by juliermefelix** [reply]
Thanks for listening.
Resolved, based on that issue 159.
https://github.com/serenity-is/Serenity/issues/159

with a little modification.
![image](https://user-images.githubusercontent.com/7516443/146864374-cbaaaf53-2e5e-43ae-96dc-f7bb6e00b50d.png)
